src/wallet/rpcwallet.cpp

```diff
@@ -37,6 +37,8 @@
 
 #include <univalue.h>
 
+#include <functional>
```

Comment:
Why is this include needed?

Reply:
According to http://en.cppreference.com/w/cpp/utility/functional/bind, `std::bind` is "[d]efined in header `<functional>`".